### ISSUE TITLE
TechEmpower PR

### ISSUE THREAD

**Issue by n-stefan** [aspnet/Benchmarks#1705]
Hi,
what is currently the recommended way of opening a pull request over at TechEmpower?
Thanks

**Comment by sebastienros** [comment]
Ideally for use if that touches the aspnet core benchmarks you could try the PRs on this repository first. That allows us to benchmark it easily and discuss about the motivations. But nobody is preventing you from creating one in TE directly if you feel confident we would also agree with the change.

**Comment by n-stefan** [comment]
I see. The PR would change the Platform Update benchmark to also use positional parameters and I've modified the SQL accordingly, however it looks like over here it no longer uses an inline temp table but a CASE statement so I'd have to redo and retest the code.

**Comment by n-stefan** [comment]
The changes for Platform Updates to use positional parameters are in the branch `positional-parameters-update` of my fork `n-stefan/Benchmarks`.
Locally it seems to boost RPS by about 3-5%, would be interesting to see how it performs on big iron.

**Comment by sebastienros** [comment]
@roji you probably have looked at it already

**Comment by roji** [comment]
Yeah, this was done in #1702 for most of the platform benchmarks, and in #1753 for the updates benchmark.

IIRC I still need to do this for the non-platform benchmarks though... Am already tracking that on my side, we can close this.
